### ISSUE TITLE
Error.

### ISSUE THREAD

**Issue by Nevercold** [TheGameSpider/TechnicSolder#36]
**Describe the bug**
Error 500 in api/index.php

The lines in api/index.php trigger an error.

```
$clientsq = mysqli_query($conn, "SELECT * FROM `clients` WHERE `id` IN (".$modpack['clients'].")");
while($client=mysqli_fetch_array($clientsq)){
	array_push($clients, $client['UUID']);
}
```

If you remove them, it works fine.

Reproduce: 
https://solder.domain.de/api/modpack?include=full&k=token

**Comment by Nevercold** [comment]
One part also fixes #35

**Comment by cowpod** [comment]
I think doing this will break the client whitelist.

**Comment by Nevercold** [comment]
Yes, the client whitelist then no longer works, but it works temporarily.

**Comment by ZandercraftGames** [comment]
I don't have this issue with clients. What does the error say?

**Comment by Nevercold** [comment]
`PHP Fatal error:  Uncaught TypeError: mysqli_fetch_array(): Argument #1 ($result) must be of type mysqli_result, bool given in /api/index.php:51\nStack trace:\n#0 /api/index.php(51): mysqli_fetch_array()\n#1 {main}\n  thrown in /api/index.php on line 51
`

**Comment by cowpod** [comment]
try to reset your database - it appears that  the mysqli_query is failing to get the data.
